weave-js/src/components/PagePanelComponents/Home/Browse3/pages/ChatView/MessagePanel.tsx

```diff
@@ -7,34 +8,98 @@ import {ToolCalls} from './ToolCalls';
 import {Message} from './types';
 
 type MessagePanelProps = {
+  index: number;
   message: Message;
   isStructuredOutput?: boolean;
+  isChoice?: boolean;
```

Comment:
This does not appear to be used

Reply:
i use this later for the message panel buttons

Reply:
removed